internal/description/description.go

```diff
@@ -118,8 +118,12 @@ The name of the log file must be one of: mongodb.gz, mongos.gz, mongodb-audit-lo
 	DBUsersLong = `The dbusers command retrieves, creates and modifies the MongoDB database users in your cluster.
 Each user has a set of roles that provide access to the project’s databases. 
 A user’s roles apply to all the clusters in the project.`
-	ConfigLongDescription = `Setting API keys is optional and env variables can be used instead.
-Leaving any option empty won't override existing stored values.`
+	ConfigLongDescription = `Configure default values in a user profile.
```

Comment:
Same question around "default" causing confusion? Maybe:
```suggestion
	ConfigLongDescription = `Configure settings in a user profile.
```